dataverse-webapp/src/main/webapp/createEditDataverse.xhtml

```diff
@@ -364,14 +353,14 @@
                     <p:panel id="editFacets" styleClass="panelLayoutBlock" widgetVar="editFacets">
                         <div class="row">
                             <div class="col-sm-12 form-group">
-                                <label class="col-sm-3 control-label"
+                                <label class="col-md-3 control-label"
                                        for="facetPickListCreate">#{bundle['dataverse.facetPickList.text']}</label>
-                                <div class="col-sm-9">
+                                <div class="col-md-9">
                                     <p class="help-block">#{bundle['dataverse.facetPickList.tip']}</p>
 
                                     <label class="metadata-blocks-default" for="facetsRoot"
                                            jsf:rendered="#{CreateEditDataversePage.dataverse.owner != null}">
-                                        <h:selectBooleanCheckbox id="facetsRoot" tabindex="8"
+                                        <h:selectBooleanCheckbox id="facetsRoot" 
                                                                  styleClass="facets-blocks-default"
                                                                  value="#{CreateEditDataversePage.inheritFacetFromParent}">
```

Comment:
I can't see a checkbox for this field - only text is rendered "Use metadata fields from Root"

Reply:
Fixed.